bin/hdfcoinc/pycbc_multiifo_add_statmap

```diff
@@ -453,9 +453,11 @@ while True:
                     combined_fg_data.data[key][:]
 
     max_each_combo = {combo: sep_fg_data[combo].data['ifar'][:].argmax()
-                      for combo in all_ifo_combos}
+                      for combo in all_ifo_combos
+                      if len(sep_fg_data[combo].data['ifar'][:]) > 0}
     max_ifars = {combo: sep_fg_data[combo].data['ifar'][:][max_each_combo[combo]]
-                    for combo in all_ifo_combos}
+                 for combo in all_ifo_combos
```

Comment:
If this is updated to `for combo in max_each_combo.keys()` that would make it a bit more readable

Reply:
I've updated to use the keys and value of the previous dict to make it simpler